ci-operator/config/windup/windup-ui-tests/windup-windup-ui-tests-main__lp-interop-ocp4.12.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  to: mtr-runner
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
```

Comment:
I'd remove this line as its pointing a specific build of OCP which is not always available, removing this line should not cause any problems. I believe not including it will just use a default of relative: 0

I say that because if you set `relative: 0` then run make update it actually deletes this line for you. Which makes me think it might be the default behavior and not worth mentioning.

Reply:
Resolved in [3fd91bd](https://github.com/openshift/release/pull/36181/commits/3fd91bd72531fc6e504ff9eb26657e0278dea6d1)